app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,54 @@
+# rubocop:disable Style/ClassAndModuleChildren
+# :reek:InstanceVariableAssumption
+class Api::V1::OrganizationsController < ApplicationController
+  before_action :set_organization, only: %i[show update]
+  before_action :authenticate_user!
+  before_action :unauthorized_user, only: %i[create update]
+
+  def show
+    if @organization && current_user[:role] == 'organizer' && organization_owner?
```

Comment:
`current_user[:role] == 'organizer'` can you add check of roles in user model and just call `current_user.organizer?`